tools/github-team-user-store/GitHubTeamUserStore/GitHubTeamUserStore/RepositoryLabelGenerator.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using GitHubTeamUserStore.Constants;
+using Octokit;
+
+namespace GitHubTeamUserStore
+{
+    public class RepositoryLabelGenerator
+    {
+        // Repository name is the key, with the list of that repository's labels as the as the value
+        private static Dictionary<string, HashSet<string>> _repoLabelDict = new Dictionary<string, HashSet<string>>();
```

Comment:
Why does this need to be static? I would suggest making this a normal member and make this a class or local to a single static method. In general folks should try avoid having global static state like this.

Reply:
This will be updated with the next commit.